crates/holochain_metrics/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "holochain_metrics"
```

Comment:
Should this be `holochain_*`? We'll want to embed this in Kitsune. That's fine if everything stays in this repo but if we decide to separate Kitsune into its own repo at any point we'll have circular deps between repos and may want to move this out too. Maybe a generic name would be best for this crate? Or it can just be a `holochain_*` crate that doesn't live in the mono repo one day, that is also fine I guess

Reply:
This is only used in binaries, similar to the tracing_subscriber. Since kitsune doesn't have a binary it shouldn't need it. And the setup is simple enough if we *do* end up wanting to initialize metrics collection in an example in the kitsune repo.

Maybe I'll consider if any of this logic can be encapsulated by the initializer in the Influxive repo.